src/inc/msquic.h

```diff
@@ -648,7 +649,9 @@ typedef enum QUIC_CONNECTION_EVENT_TYPE {
     QUIC_CONNECTION_EVENT_IDEAL_PROCESSOR_CHANGED           = 9,
     QUIC_CONNECTION_EVENT_DATAGRAM_STATE_CHANGED            = 10,
     QUIC_CONNECTION_EVENT_DATAGRAM_RECEIVED                 = 11,
-    QUIC_CONNECTION_EVENT_DATAGRAM_SEND_STATE_CHANGED       = 12
+    QUIC_CONNECTION_EVENT_DATAGRAM_SEND_STATE_CHANGED       = 12,
+    QUIC_CONNECTION_EVENT_RESUMED                           = 13,
+    QUIC_CONNECTION_EVENT_RESUMPTION_TICKET_RECEIVED        = 14
```

Comment:
It might be time to reorganize these...

Reply:
Sort alphabetically, or make into an enum?

Reply:
I'd sort it by order of arrival expectation. The events that arrive first, should be higher in the list.

Reply:
I'm going to suggest doing that in a dedicated PR.